api/src/main/java/io/strimzi/api/kafka/model/connector/ListOffsets.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.connector;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.api.kafka.model.common.Constants;
+import io.strimzi.api.kafka.model.common.ResourceReference;
+import io.strimzi.api.kafka.model.common.UnknownPropertyPreserving;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Configuration for listing connector offsets.
+ */
+@Buildable(
+        editableEnabled = false,
+        builderPackage = Constants.FABRIC8_KUBERNETES_API
+)
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+@JsonPropertyOrder({"toConfigMap"})
+@EqualsAndHashCode
+@ToString
+public class ListOffsets implements UnknownPropertyPreserving {
+    private ResourceReference configMapReference;
+    private Map<String, Object> additionalProperties;
+
+    @Description("Reference to the ConfigMap where the list of offsets will be written to.")
+    @JsonProperty(value = "toConfigMap", required = true)
```

Comment:
Why don't you just name it `toConfigMap` / `getToConfigMap` / `setToConfigMap`?

Reply:
I was trying to provide a more descriptive method name just to make the code more readable, happy to change it though if you think it's better to stay closer to the actual property name

Reply:
I personally find it much better when the names follow the API names. It makes it easiest to align the Java and the CRD models, it makes lot less confusion when checking some YAML against the Java code etc. I also think we usually don't map the property like this unless it conflicts with some Java keywords. So I personally would change this.

Reply:
Alternatively, I personally would be also fine if we change the API against the proposal and name the field simply `configMap`. It might make the name less weird and possibly allow to reuse the same class for both listing and setting the offsets.